src/addon/cc-jenkins-info.js

```diff
@@ -0,0 +1,149 @@
+import '../atoms/cc-img.js';
+import '../atoms/cc-flex-gap.js';
+import '../molecules/cc-block-section.js';
+import '../molecules/cc-block.js';
+import '../molecules/cc-error.js';
+import { css, html, LitElement } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { i18n } from '../lib/i18n.js';
+import { skeletonStyles } from '../styles/skeleton.js';
+import { ccLink, linkStyles } from '../templates/cc-link.js';
+
+const infoSvg = new URL('../assets/info.svg', import.meta.url).href;
+
+const JENKINS_LOGO_URL = 'https://static-assets.cellar.services.clever-cloud.com/logos/jenkins.svg';
+const JENKINS_DOCUMENTATION = 'https://www.clever-cloud.com/doc/addons/jenkins/';
+
+/**
+ * A component to display various informations (Documentation, access, updates, ...) for a Jenkins service.
+ *
+ * 🎨 default CSS display: `block`
+ * <br>
+ * 🧐 [component's source code on GitHub](https://github.com/CleverCloud/clever-components/blob/master/src/addon/cc-jenkins-info.js)
+ *
+ * ## Type definitions
+ *
+ * ```js
+ * interface Versions {
+ *   current: String,
+ *   available: String
+ * }
+ * ```
+ *
+ * ## Images
+ *
+ * | | |
+ * |-------|------|
+ * | <img src="assets/info.svg" style="height: 1.5rem; vertical-align: middle"> | <code>info.svg</code>
+ *
+ * @prop {String} jenkinsLink - Provides the HTTP link of the Jenkins service.
+ * @prop {String} jenkinsManageLink - Provides the HTTP link of the Jenkins management interface.
+ * @prop {Versions} versions - Provides the current and available version of the Jenkins add-on.
+ * @prop {Boolean} error - Display an error message.
+ */
+export class CcJenkinsInfo extends LitElement {
+
+  static get properties () {
+    return {
+      jenkinsLink: { type: String, attribute: 'jenkins-link' },
+      jenkinsManageLink: { type: String, attribute: 'jenkins-manage-link' },
+      versions: { type: Object, attribute: 'versions' },
+      error: { type: Boolean, attribute: 'error' },
```

Comment:
We don't have an automated way to do this yet but please sort props alphabetically (just put error first).